### DISCUSSION TITLE
Will Pnp Modern Search affected by SharePoint Add-In retirement in Microsoft 365

### DISCUSSION THREAD

**Discussion by fran2503hel** [microsoft-search/pnp-modern-search#3388] [Q&A] [answered]
Hello,

We're planning to Deploy Pnp Modern Search in our Microsoft 365 tenant.

But we have our doubts after reading this article : https://learn.microsoft.com/fr-fr/sharepoint/dev/sp-add-ins/retirement-announcement-for-add-ins

Will Pnp Modern Search be affect by this on Microsoft 365 tenant ?

Thank you for helping us understand the situation,

François

**Comment by wobba** [comment, marked as answer]
If you read the paragraph https://learn.microsoft.com/fr-fr/sharepoint/dev/sp-add-ins/retirement-announcement-for-add-ins#migrate-business-relevant-sharepoint-add-ins-by-using-newer-sharepoint-development-models you see that SPFx is what you should move over to. As PnP Modern Search is an SPFx solution, it is not affected in any way.
